apps/full-site-editing/full-site-editing-plugin/readme.txt

```diff
@@ -0,0 +1,41 @@
+=== Full Site Editing ===
+Contributors: automattic, get_dave, marekhrabe, mppfeiffer, obenland
+Tags: block, blocks, editor, gutenberg, page
+Requires at least: 5.0
+Tested up to: 5.2
+Stable tag: 1.0
+Requires PHP: 5.6.20
+License: GPLv2 or later
+License URI: https://www.gnu.org/licenses/gpl-2.0.html
+
+Enhances your page creation workflow within the Block Editor.
+
+== Description ==
+
+This plugin comes with a custom block to display a list of your most recent blog posts, as well as a template selector
+to give you a head start on creating new pages for your site.
+
+
+== Installation ==
+
+1. Upload the plugin files to the `/wp-content/plugins/full-site-editing` directory, or install the plugin through the WordPress plugins screen directly.
+1. Activate the plugin through the 'Plugins' screen in WordPress.
+1. Create a new page and select a template that best suits your needs.
+1. Place the "Blog Posts Listing" block anywhere you want inside the block editor.
+
+
+== Frequently Asked Questions ==
+
+= Can I use this plugin in production? =
+
+We'll be making frequent updates to the plugin as we flesh out its feature set. You're welcome to try it, just be aware that it could break after an update.
```

Comment:
"We'll be making frequent updates to the plugin as we flesh out its feature set. You're welcome to try it, just be aware that it is only designed to work on the WordPress.com environment and could break after an update."